mParticle-Leanplum/MPKitLeanplum.m

```diff
@@ -201,19 +200,61 @@ - (MPKitExecStatus *)removeUserAttribute:(NSString *)key {
     return execStatus;
 }
 
-- (MPKitExecStatus *)setUserIdentity:(NSString *)identityString identityType:(MPUserIdentity)identityType {
-    if (identityType == MPUserIdentityEmail) {
-        [self setUserAttribute:kMPLeanplumEmailUserAttributeKey value:identityString];
+- (MPKitExecStatus *)onLoginComplete:(FilteredMParticleUser *)user request:(FilteredMPIdentityApiRequest *)request {
+    MPKitExecStatus *execStatus = [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceLeanplum) returnCode:MPKitReturnCodeSuccess];
```

Comment:
I would suggest centralizing this code like in the Braze kit to avoid repeating the whole block

Reply:
I can do that for login and identify but not modify since it has a slightly different codepath. That ok?